massgov/openmass
# DP-22689: Make About section for org page

**This has a Mayflower branch: https://github.com/massgov/mayflower/pull/1461**

**Description:**
Updated the Organization Section paragraph to allow adding About paragraphs to the Content field and updated the migration deploy function to adjust the section order for Boards organization type.


**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-22689


**To Test:**
- Login and edit an Organization node.
- Click on the Content tab.
- Add an Organization Section.
- Set the title of the Organization Section.
- Expand the Content field.
- Click the “Add About” button.
- Add content to the fields and save.
- View the Organization node.
- Verify the section displays.
- Verify there is only one heading.
- Verify the social links and bios show, if they have content.


**Screenshots/GIFs:**

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
